lib/install/validate/validator.go

```diff
@@ -26,6 +26,8 @@ import (
 
 	"github.com/docker/go-units"
 
+	"strconv"
```

Comment:
It seems like this should be grouped with the other standard library imports.

Reply:
Done